.github/workflows/ci_cd.yml

```diff
@@ -13,19 +33,61 @@ env:
   PYMECHANICAL_PORT: 10000  # default won't work on GitHub runners
   PYMECHANICAL_START_INSTANCE: FALSE
   DOCKER_PACKAGE: ghcr.io/ansys/mechanical
-  DOCKER_IMAGE_VERSION: 23.1.0
   DOCKER_MECH_CONTAINER_NAME: mechanical
   MAIN_PYTHON_VERSION: '3.10'
   PACKAGE_NAME: ansys-mechanical-core
   PACKAGE_NAMESPACE: ansys.mechanical.core
   DOCUMENTATION_CNAME: mechanical.docs.pyansys.com
+  DEFAULT_REVN: '231'  # ensure this is the same as inputs.default-revn.default
+  TEST_REVN: '232'  # ensure this is the same as inputs.test-revn.default
+  DEV_REVN: '241'
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-
+  revn-variations:
+    name: Save variations of revn
+    runs-on: ubuntu-latest
+    outputs:
+      default: ${{ steps.save-versions.outputs.default }}
+      default_lic: ${{ steps.save-versions.outputs.default_lic }}
+      test: ${{ steps.save-versions.outputs.test }}
+      test_lic:  ${{ steps.save-versions.outputs.test_lic }}
+    steps:
+      - id: save-versions
```

Comment:
why are licenses part of variations? are those version specific?

Reply:
Yes, the license env variable is version specific: ANSYSCL232_DIR: /install/ansys_inc/v232/licensingclient, for example